src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -446,17 +450,26 @@ def _copy_registry_credentials(self):
                 "username": cred.username,
                 "password": cred.secret.get_secret_value(),
             }
-            eda_cred = models.EdaCredential.objects.create(
+            eda_cred, created = models.EdaCredential.objects.get_or_create(
                 name=cred.name,
-                description=cred.description,
-                managed=False,
-                credential_type=cred_type,
-                inputs=inputs_to_store(inputs),
+                defaults={
+                    "description": cred.description,
+                    "managed": True,
```

Comment:
All these converted credentials are user-created, `managed` should be `False`, right?